src/test/java/com/example/pfairplayservice/jpa/repository/MemberRepositoryIntegrationTest.java

```diff
@@ -38,202 +40,56 @@
-                .address("testad2")
-                .birthday(new Date())
-                .joinDate(new Date())
-                .level(1)
-                .preferPosition(1)
-                .phoneNumber("010")
-                .recentLoginDate(new Date())
-                .phoneNumberDisclosureOption(1)
-                .name("ryoo2")
-                .build();
-
-        TeamEntity teamEntity = TeamEntity
-                .builder()
-                .teamName("teamnametest")
-                .teamLeadMember(memberEntity0)
-                .activityAreaAddress("seoul")
-                .registrationDate(new Date())
-                .foundDate(new Date())
-                .build();
 
+        // given
+        MemberEntity memberEntity0 = TestEntityGenerator.generateMemberEntity();
+        MemberEntity memberEntity1 = TestEntityGenerator.generateMemberEntity();
+        MemberEntity memberEntity2 = TestEntityGenerator.generateMemberEntity();
 
+        TeamEntity teamEntity = TestEntityGenerator.generateTeamEntity(memberEntity0);
 
         memberRepository.save(memberEntity0);
         memberRepository.save(memberEntity1);
         memberRepository.save(memberEntity2);
         teamRepository.save(teamEntity);
 
-        Optional<TeamEntity> foundTeamEntity = teamRepository.findById(teamEntity.getTid());
-
-        MemberTeamId memberTeamId0 = MemberTeamId
-                .builder()
-                .uid(memberEntity0.getUid())
-                .tid(teamEntity.getTid())
-                .build();
-
-        MemberTeamId memberTeamId1 = MemberTeamId
-                .builder()
-                .uid(memberEntity0.getUid())
-                .tid(teamEntity.getTid())
-                .build();
-
-        MemberTeamId memberTeamId2 = MemberTeamId
-                .builder()
-                .uid(memberEntity0.getUid())
-                .tid(teamEntity.getTid())
-                .build();
-
-        MemberTeamEntity memberTeamEntity0 = MemberTeamEntity
-                .builder()
-                .memberTeamId(memberTeamId0)
-                .build();
-
-        MemberTeamEntity memberTeamEntity1 = MemberTeamEntity
-                .builder()
-                .memberTeamId(memberTeamId0)
-                .build();
-
-        MemberTeamEntity memberTeamEntity2 = MemberTeamEntity
-                .builder()
-                .memberTeamId(memberTeamId0)
-                .build();
+        MemberTeamEntity memberTeamEntity0 = TestEntityGenerator.generateMemberTeamEntity(memberEntity0.getUid(), teamEntity.getTid());
+        MemberTeamEntity memberTeamEntity1 = TestEntityGenerator.generateMemberTeamEntity(memberEntity1.getUid(), teamEntity.getTid());
+        MemberTeamEntity memberTeamEntity2 = TestEntityGenerator.generateMemberTeamEntity(memberEntity2.getUid(), teamEntity.getTid());
 
         memberTeamRepository.save(memberTeamEntity0);
         memberTeamRepository.save(memberTeamEntity1);
         memberTeamRepository.save(memberTeamEntity2);
```

Comment:
Remove all records which are saved in the given area after the `then` section (L 92) .
Because it can affect another test case.